scene/resources/packed_scene.h

```diff
@@ -135,6 +116,39 @@ class SceneState : public RefCounted {
 		int node = -1;
 	};
 
+	struct NodeData {
+		int parent = 0;
+		int owner = 0;
+		int type = 0;
+		int name = 0;
+		int instance = 0;
+		int index = 0;
+
+		struct Property {
+			int name = 0;
+			int value = 0;
+		};
+
+		Vector<Property> properties;
+		Vector<int> groups;
+	};
+
+	struct OverrideData {
+		NodePath path;
+
+		struct Property {
+			StringName name;
+			Variant value;
+		};
+
+		Vector<Property> properties;
+		Vector<int> groups;
```

Comment:
Unused.

Reply:
it probably should be, will look into this